src/components/errors/error.ts

```diff
@@ -7,6 +7,18 @@ import internalServerError from './error.500.njk';
 
 export class UserFriendlyError extends Error {}
 
+/* istanbul ignore next */
+function platformLocation(region: string): string {
```

Comment:
I think we can get rid of this. There are two options:

The `internalServerErrorMiddleware` is setup in the app.ts, where we know already the location. It is also called from some controllers with the location.

The `pageNotFoundMiddleware` is also setup in the controller.

I suggest we create some constructor for these two middlewares, where we pass the `location` to be used.

Maybe we can put the `internalServerErrorMiddleware` middleware in the `app.ctx` so that can be used from the controllers ([auth](https://github.com/alphagov/paas-admin/blob/294cb8b068c72f1153ce19958e71698d158cacaf/src/components/auth/auth.ts#L25-L26) and [terms](https://github.com/alphagov/paas-admin/blob/294cb8b068c72f1153ce19958e71698d158cacaf/src/components/terms/middleware.ts#L15-L16))